packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -835,7 +835,7 @@ class _AppBarState extends State<AppBar> {
         ?? theme.primaryIconTheme
       : widget.iconTheme
         ?? appBarTheme.iconTheme
-        ?? theme.iconTheme.copyWith(color: foregroundColor);
+        ?? theme.iconTheme.copyWith(color: foregroundColor, splashRadius: 18);
```

Comment:
This would change the default radius on app bar, which could be considered a breaking change. I don't think we want to do this, but instead just allow for users to make the splash radius smaller, no?

Reply:
Like this https://github.com/flutter/flutter/issues/64702#issuecomment-876364027?
Spotted this after than I filed this PR, if you think manual solution is the way to go then we can close this PR

Thank you for reviewing!

Reply:
This would be similar to how `useMaterial3` is handled

Reply:
I think by adding a splashRadius param to icon theme (like this PR does), then developers can simply specify the splash radius in their app bar's theme and not have to do it manually for each icon button.

As far as changing the default value, thats where I defer to @HansMuller. Hans what do you think we should do for the default behavior?